src/useAgaveLink.tsx

```diff
@@ -17,11 +17,17 @@ export const useAgaveLink = (config: UseAgaveLinkProps): UseAgaveLinkResponse =>
         }
     }, [isReadyForInitialization, config]);
 
-    const openLink = useCallback(() => {
-        if (window.AgaveLink) {
-            window.AgaveLink.openLink(config);
-        }
-    }, [config]);
+    const openLink = useCallback(
+        (overrideConfig: Partial<UseAgaveLinkProps> = {}) => {
+            if (window.AgaveLink) {
+                window.AgaveLink.openLink({
+                    ...config,
+                    ...overrideConfig,
+                });
+            }
+        },
+        [config]
+    );
```

Comment:
@jesseyay just to make sure I fully understand (as a non-TS expert), what would be an example where this would be necessary? In the example code I had for testing, I was able to omit a few of the parameters from the config without issue:

https://github.com/agave-api/react-agave-link-example/blob/master/src/App.js#L20-L34